<launch>

	       <node pkg="fanboat_ll" name="serial" output="screen" type="fanboat_serial.py"/>
	       <node pkg="joy" name="joystick" output="screen" type="joy_node"/>
	       <node pkg="lab2" name="joy_constant" output="screen" type="joy_constant"/>
	       <node pkg="lab2" name="joyAngleIntegrater" output="screen" type="joyAngleIntegrater"/>
	       <node pkg="lab2" name="arbitrator" output="screen" type="arbitrator"/>
	       <node pkg="lab2" name="angularPositionPID" output="screen" type="angularPositionPID"/>
	       <param name="pValue" type="double" value="120"/>
	       <param name="dValue" type="double" value=".11"/>
	       <param name="rightScale" type="double" value="1"/>
	       <param name="leftScale" type="double" value="1"/>
</launch>
